test/mocks/SimpleERC20Wrapper.sol

```diff
@@ -4,7 +4,6 @@ pragma solidity 0.8.21;
 import {ERC20} from "src/token/ERC20.sol";
 import {IERC20Wrapper, IERC20Metadata} from "src/interfaces/IERC20.sol";
 import "forge-std/Test.sol";
-import {console} from "forge-std/console.sol";
 
 contract SimpleERC20Wrapper is ERC20, IERC20Wrapper {
```

Comment:
Do we follow any specific pattern for mocks? Something like adding Mock word as a prefix ? Would be nice to diff somehow the prod code from the one used in tests only.

Reply:
Yeah I didn't use the Mock prefix here because it's not really a mock. It's really a very simple, but technically complete implementation of an ERC20 wrapper.

Reply:
We could also still add Mock prefix to it though, really no strong feelings on this one. Just felt wrong since it's not actually mocking anything.